src/gleam/io.gleam

```diff
@@ -73,24 +73,39 @@ if javascript {
 /// ```
 ///
 pub fn debug(term: anything) -> anything {
-  debug_print(term)
+  do_debug(term)
```

Comment:
Let's call the inspect function here and print it using the normal println function in Gleam.

Reply:
I suppose you mean ffi inspect. If we do this io will depend on string_builder. I am updating this PR to do this.

Or did you mean depending on `string.debug()`?